kubo/README.md

```diff
@@ -0,0 +1,21 @@
+## Kubo distribution for lido-oracle
+
+### Init scripts
+
+Kubo [executes
+scripts](https://github.com/ipfs/kubo/blob/fd50eb0fc385ec35cc2269646182920849b3c9b5/bin/container_daemon#L53)
+found in the `/container-init.d/` in alphabetical order.
+
+#### `./010-set-conf.sh`
+
+- Updates configuration of Kubo node to get determinstic CIDs
+- Set ports to expected by the client
```

Comment:
let's move ports config outside, so guys can configure everything in their docker-compose files

Reply:
The ports are configured via env variables already.